awa-python/tests/test_progress.py

```
@@ -11,16 +11,29 @@
     tx = await c.transaction()
-    await tx.execute("DELETE FROM awa.jobs WHERE queue LIKE 'progress_%'")
+    await tx.execute("DELETE FROM awa.queue_meta WHERE queue LIKE 'progress_%'")
     await tx.commit()
-    return c
+    try:
+        yield c
+    finally:
+        try:
+            await c.shutdown()
+        except Exception:
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)